cognite/neat/rules/models/_rules/dms_schema.py

```diff
@@ -192,6 +375,11 @@ class PipelineSchema(DMSSchema):
     databases: DatabaseWriteList = field(default_factory=lambda: DatabaseWriteList([]))
     raw_tables: RawTableWriteList = field(default_factory=lambda: RawTableWriteList([]))
 
+    _FIELD_NAME_BY_RESOURCE_TYPE: ClassVar[dict[str, str]] = {
+        **DMSSchema._FIELD_NAME_BY_RESOURCE_TYPE,
+        "raw": "raw_tables",
+    }
+
     _SQL_TYPE_BY_PROPERTY_TYPE: ClassVar[dict[type[dm.PropertyType], str]] = {
```

Comment:
I thought this was moved under `cognite.neat.rules.modesl._rules._types._value`, to leave with other `_value` types.

Reply:
Sorry, forgot to do it, will open an separate PR for it.